packages/stellar-axelar-std-derive/src/storage.rs

```diff
@@ -142,21 +134,26 @@ impl Value {
     fn status_fns(
         &self,
         storage_type: &StorageType,
-        getter_name: &Ident,
-        setter_name: &Ident,
-        remover_name: &Ident,
+        FnNames {
+            getter_name,
+            setter_name,
+            remover_name,
+            ttl_extender_name,
+            ..
+        }: &FnNames,
         params: &TokenStream,
         storage_key: &TokenStream,
     ) -> TokenStream {
         let storage_method = storage_type.storage_method();
         let ttl_fn = storage_type.ttl_fn(&quote! { key });
```

Comment:
```suggestion
        let ttl_fn = storage_type.ttl_fn(&quote! { key }, &quote! { threshold }, ...);
```
parameterize the variable names